### ISSUE TITLE
FE restart error

### ISSUE THREAD

**Issue by stalary** [apache/incubator-doris#3355]
**Describe the bug**
Fe died, after restart the following error

Deployment use one fe pattern

**Screenshots**
```
2020-04-20 16:12:58,123 WARN 1 [Catalog.notifyNewFETypeTransfer():2264] notify new FE type transfer: UNKNOWN
2020-04-20 16:13:00,133 WARN 56 [Catalog.notifyNewFETypeTransfer():2264] notify new FE type transfer: MASTER
2020-04-20 16:13:11,432 WARN 67 [Catalog.replayJournal():2418] replay journal cost too much time: 11285 replayedJournalId: 211440
2020-04-20 16:13:21,552 ERROR 66 [EditLog.logEdit():812] Fatal Error : write stream Exception
java.lang.NullPointerException: null
	at org.apache.doris.transaction.TransactionState.write(TransactionState.java:563) ~[palo-fe.jar:?]
	at org.apache.doris.journal.JournalEntity.write(JournalEntity.java:118) ~[palo-fe.jar:?]
	at org.apache.doris.journal.bdbje.BDBJEJournal.write(BDBJEJournal.java:145) ~[palo-fe.jar:?]
	at org.apache.doris.persist.EditLog.logEdit(EditLog.java:810) [palo-fe.jar:?]
	at org.apache.doris.persist.EditLog.logDeleteTransactionState(EditLog.java:1170) [palo-fe.jar:?]
	at org.apache.doris.transaction.GlobalTransactionMgr.deleteTransaction(GlobalTransactionMgr.java:263) [palo-fe.jar:?]
	at org.apache.doris.transaction.GlobalTransactionMgr.removeExpiredAndTimeoutTxns(GlobalTransactionMgr.java:869) [palo-fe.jar:?]
	at org.apache.doris.catalog.Catalog$2.runAfterCatalogReady(Catalog.java:2178) [palo-fe.jar:?]
	at org.apache.doris.common.util.MasterDaemon.runOneCycle(MasterDaemon.java:58) [palo-fe.jar:?]
	at org.apache.doris.common.util.Daemon.run(Daemon.java:116) [palo-fe.jar:?]
```

**Comment by morningman** [comment]
What is your Doris version?

**Comment by stalary** [comment]
> What is your Doris version?

master

**Comment by morningman** [comment]
I know, I will fix it.

**Comment by imay** [comment]
Hi @morningman , does this bug exist in 0.12 version? 
If it does, need to describe when this bug will be triggered and is there a way to avoid it.
 If the users have nothing else to do but upgrade, then we may have to release 0.12.1

**Comment by morningman** [comment]
> Hi @morningman , does this bug exist in 0.12 version?
> If it does, need to describe when this bug will be triggered and is there a way to avoid it.
> If the users have nothing else to do but upgrade, then we may have to release 0.12.1

Not included, this bug is introduced by #3293
